api/systemIntegrations/put.js

```diff
@@ -246,6 +243,14 @@ function _postProvider(bag, next) {
     name = name.replace('Keys', '');
   else if (name.endsWith('BasicAuth'))
     name = name.replace('BasicAuth', '');
+  bag.providerName = name;
+
+  if (!bag.reqBody.data.url) {
```

Comment:
Shouldn't these checks happen in checkInputParams?

Reply:
this is added here because initially we create the sysInt without any url. so if the url is not present then we dont create the provider. If it is present, then we create the provider.